sydent/pipeline.yml

```diff
@@ -5,3 +5,18 @@ steps:
     plugins:
       - docker#v3.0.1:
           image: "matrixdotorg/sydent-build"
+  - command:
+      - "echo '--- Install dependencies'"
+      - "pip install -e ."
+      - "echo '--- Install matrix-is-tester'"
+      - "python -m pip install git+https://github.com/matrix-org/matrix-is-tester.git"
```

Comment:
why do we `pip install` above and `python -m pip install` here?

Reply:
uh, probably just copied from somewhere